File: baekjoon/2500_2599/2577/2577.py
import sys
input = sys.stdin.readline

a, b, c = [int(input()) for i in range(3)]
multiply = a * b * c
arr = [0] * 10

while multiply > 0:
    temp = multiply % 10
    arr[temp] += 1
    multiply //= 10

for i in range(10):
    print(arr[i])

File: baekjoon/2500_2599/2577/2577.c
#include <stdio.h>

int main(){
  int a, b, c, multiply, temp;
  int arr[10] = {0, };
  scanf("%d %d %d", &a, &b, &c);
  multiply = a * b * c;

  while(multiply>0){
    temp = multiply % 10;
    arr[temp]++;
    multiply /= 10;
  }

  for(int i=0; i<=9; i++){
    printf("%d\n", arr[i]);
  }

  return 0;
}
